make lint error

Thanks for submitting a bug report, please fill in the following details.

**Description**

There is a linting error around the usage of `.md` constant.


**Reproduction Steps**

Steps to reproduce the behavior:
1. Install go
1. Install goimports
1. Install other two required libraries
1. Run `make lint` on `main`.
1. See error

**What happened**

```sh
~/src/changie$ make lint
goimports -w -local github.com/miniscruff/changie .
golangci-lint run ./...
core/utils_test.go:285:54: string `.md` has 6 occurrences, make it a constant (goconst)
					&then.MockDirEntry{MockName: tc.latestVersion + ".md"},
```

**Expected behavior**

Clean linting command

**Additional context**

This is on a Mac and I installed everything from `brew`.

`go version go1.21.3 darwin/arm64`
`go install golang.org/x/tools/cmd/goimports@latest`

--- a/.github/workflows/changelog-check.yml
+++ b/.github/workflows/changelog-check.yml
@@ -1,5 +1,3 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Require change fragment
 
 on:

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -40,7 +40,7 @@ jobs:
       with:
         # Required: the version of golangci-lint is required and must be specified
         # without patch version: we always use the latest patch version.
-        version: v1.54
+        version: v1.55
 
     - name: Gen
       run: go run main.go gen

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -4,7 +4,7 @@ linters-settings:
   gci:
     local-prefixes: github.com/miniscruff/changie
   goconst:
-    min-len: 2
+    min-len: 5
     min-occurrences: 5
   gocyclo:
     min-complexity: 16
@@ -17,7 +17,6 @@ linters-settings:
   misspell:
     locale: US
   nolintlint:
-    allow-leading-space: true # don't require machine-readable nolint directives (i.e. with no leading space)
     allow-unused: false # report any unused nolint directives
     require-explanation: false # don't require an explanation for nolint directives
     require-specific: false # don't require nolint directives to be specific about which linter is being skipped

